package.json

```diff
@@ -19,43 +19,42 @@
     "ionic_webpack": "webpack.config.js"
   },
   "dependencies": {
```

Comment:
We dropped some `^`s here, mostly in Ionic plugins. Should we try being a smidge less rigid, or does it not matter?